src/components/WhyTechSVG/HandsSVG.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import handsSVG from '../../images/about-us/why-tech/hands.svg';
+
+export default function HandsSVG(){
+    return <img style={{width: '300px', height: '300px'}} alt="city" src={handsSVG} />
```

Comment:
can we directly use <svg> here rather than <img> so that we can ;et svg handle the responsivenss (otherwise img width/height would need to be managed)